README.md

```diff
@@ -1,16 +1,3 @@
-Chitter Challenge
-=================
-
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
-
-Challenge:
--------
-
-As usual please start by forking this repo.
-
 We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
```

Comment:
For future - add info on how to create the db and db/migrations directories with sql queries